### ISSUE TITLE
Window is not centered on DesktopGL in HiDPI Windows environment

### ISSUE THREAD

**Issue by neofuji** [MonoGame/MonoGame#5092]
Confirmed on 125 % environment.

**Comment by Sizaar** [comment]
The same thing happens under DX with a non-HiDPI screen, when you try to change the size of the window.

**Comment by mrhelmut** [comment]
On which OS did you encountered this? To get HiDPI right on Windows, you should add a manifest file to enable it with this in it:
```
  <asmv3:application xmlns:asmv3="urn:schemas-microsoft-com:asm.v3">
    <asmv3:windowsSettings xmlns="http://schemas.microsoft.com/SMI/2005/WindowsSettings">
      <dpiAware>true</dpiAware>
    </asmv3:windowsSettings>
  </asmv3:application>
```

On MacOS, your game must be bundled without any HighResolution keys in its Info.plist (i.e. Retina support should be disabled for MonoGame apps to correctly handle DPI on their own).

**Comment by KonajuGames** [comment]
> To get HiDPI right on Windows, you should add a manifest file to enable it with this in it:

Has that been added to our Windows project templates yet?

**Comment by neofuji** [comment]
I confirmed it on Windows.

**Comment by mrhelmut** [comment]
>

Not yet, but adding one with the compatibility and privilege fields in as well would be the way to go, I think:

```
<?xml version="1.0" encoding="utf-8"?>
<asmv1:assembly manifestVersion="1.0" xmlns="urn:schemas-microsoft-com:asm.v1" xmlns:asmv1="urn:schemas-microsoft-com:asm.v1" xmlns:asmv2="urn:schemas-microsoft-com:asm.v2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
  <assemblyIdentity version="1.0.0.0" name="AssemblyName"/>
  <trustInfo xmlns="urn:schemas-microsoft-com:asm.v2">
    <security>
      <requestedPrivileges xmlns="urn:schemas-microsoft-com:asm.v3">
        <requestedExecutionLevel level="asInvoker" uiAccess="false" />
      </requestedPrivileges>
    </security>
  </trustInfo>
  <compatibility xmlns="urn:schemas-microsoft-com:compatibility.v1">
    <application>
      <!-- Windows Vista -->
      <supportedOS Id="{e2011457-1546-43c5-a5fe-008deee3d3f0}" />
      <!-- Windows 7 -->
      <supportedOS Id="{35138b9a-5d96-4fbd-8e2d-a2440225f93a}" />
      <!-- Windows 8 -->
      <supportedOS Id="{4a2f28e3-53b9-4441-ba9c-d69d4a4a6e38}" />
      <!-- Windows 8.1 -->
      <supportedOS Id="{1f676c76-80e1-4239-95bb-83d0f6d0da78}" />
      <!-- Windows 10 -->
      <supportedOS Id="{8e0f7a12-bfb3-4fe8-b9a5-48fd50a15a9a}" />
    </application>
  </compatibility>
  <asmv3:application xmlns:asmv3="urn:schemas-microsoft-com:asm.v3">
    <asmv3:windowsSettings xmlns="http://schemas.microsoft.com/SMI/2005/WindowsSettings">
      <dpiAware>true</dpiAware>
    </asmv3:windowsSettings>
  </asmv3:application>
</asmv1:assembly>
```

Sorry, I don't have much time for PRs lately, we're finishing our game for the Gamescom.

**Comment by Jjagg** [comment]
> Sorry, I don't have much time for PRs lately, we're finishing our game for the Gamescom.

@mrhelmut I'll try this and set up a PR if you don't mind. I've been moving around my local changes for a while to be able to run DX projects, so I'd like to get this fixed.

**Comment by mrhelmut** [comment]
That would be very welcome. :) If you could sneak the same change to the DesktopGL template, that would be perfect. Mind the ```name="AssemblyName"```.

**Comment by MichaelDePiazzi** [comment]
> The same thing happens under DX with a non-HiDPI screen, when you try to change the size of the window.

I raised issue #4740 about this DX issue. @tomspilman was gonna sort it out, but I guess he got sidetracked. Not sure if it's related to this GL issue though.

**Comment by mrhelmut** [comment]
I believe the GL issue will be fixed by #5096, but I can't be sure about the DX code path, so better following up #4740 then.
